packages/ag-charts-community/src/scene/group.ts

```diff
@@ -219,19 +228,6 @@ export class Group<D = any> extends Node<D> {
         super.render(childRenderCtx); // Calls markClean().
     }
 
-    private skipRender(childRenderCtx: RenderContext) {
-        const { stats } = childRenderCtx;
-
-        for (const child of this.children()) {
-            // Skip invisible children, but make sure their dirty flag is reset.
-            child.markClean();
```

Comment:
The intent is that all child `dirty` flags are reset at this point, as we have had a recurring class of bug where unrendered nodes never had their dirty flags reset to `false` - we need to be sure we're not allowing this class of issue to be re-introduced.

Reply:
If something gets marked, all its parents get marked dirty. Only groups track dirty state. Groups may skip rendering if they're not dirty - which means none of their children are dirty. A group's dirty flag is reset on render